docs/resources/integration.md

```diff
@@ -0,0 +1,54 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "juju_integration Resource - terraform-provider-juju"
+subcategory: ""
+description: |-
+  A resource that represents a Juju Integration.
+---
+
+# juju_integration (Resource)
+
+A resource that represents a Juju Integration.
+
+## Example Usage
+
+```terraform
+resource "juju_integration" "this" {
+  model = "development"
+
+  application {
+    name        = "wordpress"
+    integration = "db" # can be optional
```

Comment:
```suggestion
    endpoint = "db" # can be optional
```

@jameinel @juanmanuel-tirado I think `endpoint` here would be more appropriate?

Reply:
Indeed `endpoint` is more appropriate here.